com.bluejeans.BlueJeans.yaml

```diff
@@ -16,8 +16,9 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --persist=.config
-  - --persist=.macromedia
+  - --persist=.pki
```

Comment:
And do you know what this was required by?

Reply:
These directories are all created/used by the Blue Jeans client. The .macromedia one was used by Flash that is not present anymore in the BJ2 client. The .pki one is used by BJ2, probably to store some certificates/keys.